crates/bevy_mesh/src/primitives/extrusion.rs

```diff
@@ -72,7 +72,7 @@ impl PerimeterSegment {
 /// ## Warning
 ///
 /// By implementing this trait you guarantee that the `primitive_topology` of the mesh returned by
-/// this builder is [`PrimitiveTopology::TriangleList`](wgpu::PrimitiveTopology::TriangleList)
+/// this builder is [`PrimitiveTopology::TriangleList`](crate::PrimitiveTopology::TriangleList)
```

Comment:
```suggestion
/// this builder is [`PrimitiveTopology::TriangleList`](PrimitiveTopology::TriangleList)
```

(Nit) The `crate` shouldn't be necessary, since you import `PrimitiveTopology` already.